partcad/src/partcad/ai_feature_file.py

```
@@ -81,12 +83,12 @@ def include_content(match: re.Match):
                 location = location[1:-1]
             try:
                 content_inserts.append(AiContentFile(location))
-                return "CONTENT_INSERTED_HERE"
+                return MARKER
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)